README.md

```diff
@@ -114,133 +149,109 @@ Install eigenlayer-cli:
 make install_eigenlayer_cli
 ```
 
-### Keystores
+### Booting Devnet with Default configs
 
-To create a keystore, run:
-
-```bash
-cast wallet new-mnemonic
-cast wallet import <keystore-name> --private-key <private-key>
-```
+Before starting you need to setup an S3 bucket. More data storage will be tested in the future.
 
-To create an ECDSA keystore, run:
+You need to fill the data in:
 
-```bash
-eigenlayer operator keys import --key-type ecdsa <keystore-name> <private-key>
-```
+```batcher/.env```
 
-To create a BLS keystore, run:
+And you can use this file as an example on how to fill it:
 
-```bash
-eigenlayer operator keys import --key-type bls <keystore-name> <private-key>
-```
+```batcher/.env.example```
 
-## How to setup a local devnet with Anvil
+After having the env setup, run in different terminals the following commands to boot Aligned locally:
 
 ```bash
 make anvil_start_with_block_time
 ```
 
-To start sending proofs, in another terminal window:
-
 ```bash
 make aggregator_start
 ```
 
-In another terminal window:
-
-```bash
-make operator_full_registration
-```
-
 ```bash
-make operator_start
+make operator_register_and_start
```

Comment:
should we add some directives too for the case where the operator is already registered and has to be restarted? It could be something like:

---

The first time booting a new operator, it needs to be registered with EigenLayer before starting it. To do this all in one shot run

```bash
make operator_register_and_start
```
**Optional**: If for any reason you want to restart the operator and the local anvil node is still running, it doesn't need to be re-registered. If you do so, an error will be prompted telling that the operator is already registered. 
In this is the case, simply run 

```bash
make operator_start
```

Reply:
This is smart, I added this